apps/wallet-mobile/src/features/Discover/common/LabelCategoryDApp.tsx

```diff
@@ -44,7 +41,7 @@ const useStyles = () => {
   })
 
   const colors = {
-    gradientColor: color.bg_gradient_1,
+    gradientBg: ['#17D1AA', '#1ACBAF', '#21B8BC', '#2E9BD3', '#3F71F1', '#475FFF'],
```

Comment:
@michaeljscript can you check with Dima if this gradient is not in Figma and why?

Reply:
Okie, I'm checking